src/helpers/figures/charts/scorecard_chart_config_builder.ts

```diff
@@ -269,37 +368,27 @@ class ScorecardChartConfigBuilder {
           this.secondaryFontColor,
       },
       baselineDescr: {
-        font: getDefaultContextFont(baselineFontSize * BASELINE_DESCR_FONT_RATIO),
+        font: getDefaultContextFont(baselineDescrFontSize),
+        splitted: splittedBaseline,
```

Comment:
`splitted` does not appear to be used in english. Could you rename all variables where it's used?

https://english.stackexchange.com/questions/467873/the-past-participle-of-split-split-or-splitted#:~:text=Splitted%20appears%20to%20be%20a,past%20participle%20of%20the%20verb.&text=Google%20Books%20shows%20very%20few%20usage%20instances%20of%20splitted%20compared%20to%20split.
```suggestion
        isSplit: isBaselineSplit,
```